exercises/custom-set/src/main/kotlin/CustomSet.kt

```diff
@@ -0,0 +1,40 @@
+class CustomSet(private var set: Set<Int> = setOf()) {
```

Comment:
Test set never requires constructing our class from a set. It's ok to use this two-constructor approach in ref implementation, but we should not demand it from a student, as it is not a part of spec. 

It looks like only `vararg`-based constructor is strictly required. And I'd better hide this clue to avoid a student taking it for granted. `Vararg` is definitely a concept a student must pay attention to.

Let's stub it lightly like 
```
class CustomSet {
    // TODO: implement proper constructor
    ...
}
```